apps/coscrad-frontend/src/utils/generic-components/presenters/generic-index-presenter.tsx

```diff
@@ -0,0 +1,19 @@
+import { IBaseViewModel, IIndexQueryResult } from '@coscrad/api-interfaces';
+import { Link } from 'react-router-dom';
+import { FunctionalComponent } from '../../types/functional-component';
+
+export const GenericIndexPresenter: FunctionalComponent<IIndexQueryResult<IBaseViewModel>> = ({
```

Comment:
Be sure to reuse this for whatever resources currently have a hard wired index view placeholder.

Reply:
Geoff will be doing this.